volume/drivers/fake/fake.go

```diff
@@ -51,6 +51,7 @@ type driver struct {
 	volume.QuiesceDriver
 	volume.CredsDriver
 	volume.CloudBackupDriver
+	volume.CloudMigrateDriver
```

Comment:
Implement an in-memory implementation of this in the fake driver.

Reply:
I don't see any value in adding that here. The implementation should test it.

Reply:
Need the implementation in fake driver so that when this is added to the SDK, the user of the SDK can use the mock-sdk-server container and test their code. If this is not added at the time of creation, it will must likely never get added by the original author.

Reply:
This still needs to be done.